.github/workflows/android_arm64.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: aks-linux-16-cores
+    runs-on: aks-linux-16-cores-32gb
```

Comment:
what's about releases/2023/3? 
do we need to port such updates? I suppose you will drop old labels after this PR being merged

Reply:
Yep, these things should be back-ported to all active branches